web/src/lib/components/asset-viewer/photo-viewer.svelte

```diff
@@ -87,10 +88,16 @@
 
   zoomImageWheelState.subscribe((state) => {
     photoZoomState.set(state);
+
+    if (state.currentZoom > 1 && isImageWebCompatible(asset) && !hasOriginalDataFetchingBeenTriggered) {
+      hasOriginalDataFetchingBeenTriggered = true;
+      loadAssetData({ loadOriginal: true });
```

Comment:
What happens if the original image fails to load? Ideally, it should continue to show the preview image.

Reply:
Yes, that's exactly what happens. In case of failed request, the `assetData` variable is not overriden and it continue to show the preview image.

Reply:
Perfect!